webpack/assets/javascripts/react_app/common/helpers.js

```diff
@@ -51,6 +51,24 @@ export const clearSpaces = string => string.trim().replace(/\s\s+/, ' ');
 export const getDisplayName = Component =>
   Component.displayName || Component.name || 'Component';
 
+export const urlBuilder = (controller, action, id = undefined) =>
+  `/${controller}/${id ? `${id}/` : ''}${action}`;
+
+export const urlWithSearch = (base, searchQuery) =>
+  `/${base}?search=${searchQuery}`;
```

Comment:
is this is required? /cc @Laviro

Reply:
In the `AutoComplete` I've used the `URI` library,
for example:
```
const APIPath = new URI(url);
  APIPath.addSearch({ search: APISearchQuery });
  return APIPath.toString();
```